core/Command/User/AddAppPassword.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```

Comment:
This doesn't seem right

Reply:
How about this then?
```suggestion
 * @copyright Copyright (c) 2020, NextCloud, Inc.
```